electrumx/server/block_processor.py

```diff
@@ -543,11 +543,12 @@ def validate_ft_rules_raw_tx(self, raw_tx):
     
     # Query general data including the cache
     def get_general_data_with_cache(self, key):
-        try:
-            return self.general_data_cache[key]
-        except KeyError:
-            return self.db.get_general_data(key)
-        return None 
+        cache = self.general_data_cache[key]
```

Comment:
I think use `key in self.general_data_cache:` and then get the key value will be better?
if the key not in cache, might except KeyError.  alse can use `get` function

Reply:
Oh good spot, this is an accident.

Reply:
> > I think use `key in self.general_data_cache:` and then get the key value will be better?
> 
> The main idea I applied is to avoid duplicate reading.

so, use `get` will be better.

Reply:
Fixed.